Code/GraphMol/RWMol.cpp

```diff
@@ -448,4 +466,36 @@ unsigned int RWMol::finishPartialBond(unsigned int atomIdx2, int bondBookmark,
   return addBond(bsp->getBeginAtomIdx(), atomIdx2, bondType);
 }
 
+void RWMol::beginBatchEdit() {
+  if (dp_delAtoms || dp_delBonds) {
+    BOOST_LOG(rdWarningLog) << "batchEdit mode already enabled, ignoring "
```

Comment:
I think this should be an exception.  Something is wrong in the code and you should be forced to with this as an error.